android-auto/README.md

```diff
@@ -19,14 +19,27 @@ Testing on a car, requires us to release a version of one tap and then getting a
 
 1. Install the emulator: SDK Manager > SDK Tools > Android Auto Desktop Head Unit Emulator > **install**
 1. Make sure you have the Android Auto app on your phone https://play.google.com/store/apps/details?id=com.google.android.projection.gearhead
-1. Android Auto App, enable developer settings by tapping on device info and then version info
+1. Android Auto App, enable developer settings (of the Android Auto application) by tapping on version info (several times) under "About"
 1. Android Auto App > click the hamburger on the top right > Start head unit server
 1. Set the `ANDROID_HOME` environment variable to your android SDK location (e.g., /Users/{user}/Library/Android/sdk)
-1. $ make car
+1. $ make car (this does not work when "make" file is not present in current directory) --> TODO: Add Make file, or follow below commands
```

Comment:
The make command comes from 1tap. Instead of saying something like this, it may be better to add a Makefile to this examples repo

https://github.com/mapbox/1tap-android/blob/f6a351ef0ba6ae70843fc12026172abae6e3a3b8/Makefile#L113-L118